v24/src/main/resources/kbl24/KBL24_SR1-ext.xjb

```diff
@@ -27,48 +27,43 @@
 	</jxb:bindings>
 
 	<jxb:bindings multiple="true" node="//xs:complexType[.//xs:element[@name='Id']]">
```

Comment:
This was overlooked: https://github.com/4Soft-de/kbl-model/pull/51/files#r810127222

Reply:
What do you mean?

Reply:
> This file uses both spaces and tabs. The formatting of `<ci:code>` in line 21 could also be improved.

GitHub sadly neither shows all comments per default nor can you link those comments in any way since it is not shown..

Reply:
not in this branch